raiden/tests/smart_contracts/test_netting_channel.py

```diff
@@ -1078,6 +1078,101 @@ def test_unlock_expired_lock(reveal_timeout, tester_channels, tester_state):
         )
 
 
+@pytest.mark.parametrize('settle_timeout', [50])
+@pytest.mark.parametrize('reveal_timeout', [5])
+def test_unlock_both_participants(
+        deposit,
+        settle_timeout,
+        reveal_timeout,
+        tester_channels,
+        tester_state,
+        tester_token):
+
+    pkey0, pkey1, nettingchannel, channel0, channel1 = tester_channels[0]
```

Comment:
Optional suggestion: Can we be naming the participants somehow different? It's difficult to follow the tests with `0` and `1`. Even `Alice` and `Bob` may be better to read. Or `closer`, `counterparty` e.t.c.